Tools/autotest/param_metadata/param_parse.py

```diff
@@ -480,9 +485,9 @@ def validate(param):
     # only generate rst for SITL for now:
     emitters_to_use = ['rst']
 
-# actually invoke each emiiter:
+# actually invoke each emitter:
 for emitter_name in emitters_to_use:
-    emit = all_emitters[emitter_name](sitl=args.emit_sitl)
+    emit = all_emitters[emitter_name](sitl=args.emit_sitl, latexpdf=args.emit_rst_latex)
```

Comment:
This is not what I was expecting.

I was expecting all_emitters to have another entry, 'LatexPDFRSTEmit` or similar, that being defined as

```
class LatexPDFRSTEmit(RSTEmit):
```

and overriding just the functionality it needs to.

Reply:
Done. Naming might be slightly off. But it works as expected.